Autopilot/PathManager/Src/landingMode/landingModeStages.cpp

```diff
@@ -0,0 +1,416 @@
+#include "landingModeStages.hpp"
+#include "SensorFusion.hpp"
+
+/***********************************************************************************************************************
+ * Static Member Variable Declarations
+ **********************************************************************************************************************/
+
+_LandingTakeoffInput LandingTransitionStage::_input;
+_LandingTakeoffOutput LandingTransitionStage::_output;
+_WaypointManager_Data_In LandingTransitionStage::_waypoint_input;
+_WaypointManager_Data_Out LandingTransitionStage::_waypoint_output;
+_WaypointStatus LandingTransitionStage::_waypoint_status;
+_PathData* LandingTransitionStage::_current_location;
+_PathData* LandingTransitionStage::_path_array[3];
+WaypointManager LandingTransitionStage::_landing_path;
+_LandingPath LandingTransitionStage::_path;
+
+_LandingTakeoffInput LandingSlopeStage::_input;
+_LandingTakeoffOutput LandingSlopeStage::_output;
+_WaypointManager_Data_In LandingSlopeStage::_waypoint_input;
+_WaypointManager_Data_Out LandingSlopeStage::_waypoint_output;
+
+_LandingTakeoffInput LandingFlareStage::_input;
+_LandingTakeoffOutput LandingFlareStage::_output;
+_WaypointManager_Data_In LandingFlareStage::_waypoint_input;
+_WaypointManager_Data_Out LandingFlareStage::_waypoint_output;
+
+_LandingTakeoffInput LandingDecrabStage::_input;
+_LandingTakeoffOutput LandingDecrabStage::_output;
+_WaypointManager_Data_Out LandingDecrabStage::_waypoint_output;
+
+_LandingTakeoffInput LandingTouchdownStage::_input;
+_LandingTakeoffOutput LandingTouchdownStage::_output;
+_WaypointManager_Data_Out LandingTouchdownStage::_waypoint_output;
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
+
+void LandingTransitionStage::execute(LandingMode* land_mode) {
+    // Reset output data
+    land_mode->resetPassby(&_output.controlDetails);
```

Comment:
this should be done outside of landing, since takeoff and cruising needs it as well

Reply:
yeah, this exists in the current devel repo, maybe rebase?